src/domain/event/enrol/enrol.module.scss

```diff
@@ -0,0 +1,62 @@
+@import 'colors';
+@import 'variables';
+@import 'fonts';
+@import 'mixins';
+
+.wrapper {
+  @include background($hel-light);
+  display: grid;
+  justify-items: center;
+  grid-template-rows: min-content;
+  @include respond-above(sm) {
+    padding: $largePadding;
+  }
+}
+
+.enrolContainer {
+  @include background($hel-white);
+}
+
+.enrolWrapper {
+  @include background($hel-white);
+  padding: $basePadding;
+  display: grid;
+  justify-items: center;
+}
+
+.heading {
+  text-align: center;
+}
+
+.occurrenceInfo {
+  @include background($hel-light);
+  padding: $basePadding;
+  width: 100%;
+  display: grid;
+  grid-template-rows: 2;
+  grid-template-columns: 4;
+  & > * {
+    display: inline;
+  }
+}
```

Comment:
Obsolete?

Reply:
Not exactly, but I refactored the OccurrenceInfo component to take a className as parameter

Reply:
Got it!